### DISCUSSION TITLE
How to install on Debian/Ubuntu?

### DISCUSSION THREAD

**Discussion by NicolasGoeddel** [input-leap/input-leap#1675] [Q&A] [answered]
### What happened?

In comparison to the Barrier project there seems to be no DEB package out there to install input-leap. I also did not find a snap or flatpak package.

There is also no documentation on how to install it without such a package, and the release page is empty. There are just a few tags which don't help me either.

The only packages I could find, are these mentioned in the README: https://repology.org/project/input-leap/versions

There are other issues regaring debian packages but there were migrated from the Barrier project and don't help in any way.

So how can I switch from Barrier to input-leap on Ubuntu then? :thinking: 

Thank you!

### Version

v2.4.0

### If applicable, where did you install Input Leap from?

I did not install it.

### What OSes are you seeing the problem on? (Check all that apply)

Linux

### What OS versions are you using?

Ubuntu 22.04.2 LTS

**Comment by shymega** [comment, marked as answer]
(I converted this to a Q/A because issues are for _bug reports_ or feature requests. Questions should belong in this area.)

No Debian, or other packages yet. Even using from Git in production isn't a good idea.

You can 'watch' the repository in the 'releases' category, and you should get notifications when v3.0.0 is released.

**Reply by NicolasGoeddel** [reply]
Thanks!

What's the roadmap to 3.0.0? What can I expect from it?

And am I right to use barrier in version 2.4.0 until then? So far I have no issues with it besides some missing configuration things I'd like to have.

**Reply by shymega** [reply]
@NicolasGoeddel 

There is a milestone, but most of the changes are internal. There are no real UI changes, apart from a potential bump to Qt 6. This hasn't been merged yet.

You can remain using Barrier/Input Leap (from source), on version v2.4.0 - or failing that, at _least_ v2.3.4, as they have the vulns patched.

**Reply by NicolasGoeddel** [reply]
How is it going today? When can we expect a new release?

**Reply by shymega** [reply]
Not yet.

**Reply by bittcrak** [reply]
> Not yet.

mom, are we there yet

**Reply by teelittle** [reply]
one year later - knowing you do this for fun or following your own needs - I'd like to bump this topic and remind that DEB packages do matter for a potentially large user base. 
I am not a coder, but AFAIK creating a DEB package is no more complicated that building RPM or others. 
At least for the ./debian metadata subdirectory needed, the outdated "barrier" repo could serve as a blueprint, couldn't it?

**Reply by nbolton** [reply]
There is a relevant discussion that may be work a look:
- #2199

The project mentioned produces both .deb and .rpm packages.
